saito-wasm/src/wasm_io_handler.rs

```diff
@@ -162,6 +163,11 @@ impl InterfaceIO for WasmIoHandler {
         "data/blocks/".to_string()
     }
 
+    fn create_block_directory(&self) -> std::io::Result<()> {
```

Comment:
did this work? this should not contain any IO code. it should be handled from js code